lucifer/Shell/__init__.py

```diff
@@ -0,0 +1,61 @@
+import lucifer.Indexing as Indexing
+from lucifer.Help import help_menu
+
+
+class Shell:
+    from ._In import parseShellIn, getIn
+    from ._Info import help, print_name, print_id, print_auto_vars
+    from ._Spawn import spawn_shell, spawn
+    from ._Options import show, show_options, command_set, change_auto_set_vars
+    from ._Module import describe_module, run_module, use_module, set_vars, use
+    from ._Shell import open_shell, show_shells, set_name, set_name_id, clear_shell, background_shell
+
+    def __init__(self, ID, lucifer_manager):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/manual/alexspam9090/Lucifer/pullRequest?prid=6170630)